kale/pipeline/protonet.py

```diff
@@ -0,0 +1,121 @@
+# =============================================================================
+# Author: Wenrui Fan, wenrui.fan@sheffield.ac.uk
```

Comment:
Be aware your shef email may not be permanent.

Reply:
Changed to private email.